web/views/component_view.ex

```diff
@@ -31,6 +31,45 @@ defmodule Healthlocker.ComponentView do
     day <> "/" <> month <> "/" <> year
   end
 
+  @tips ["Connect", "KeepLearning", "GiveToOthers", "BeActive", "TakeNotice"]
+
+  def markdown(body) do
```

Comment:
We use markdown in other templates (e.g. post) but this does not need anything for `wrap_tags` at the moment.  It would be good to have that moved in here as well, but may need to differentiate between how it works for this, and all other uses.